superagi/models/agent_config.py

```diff
@@ -32,3 +34,50 @@ def __repr__(self):
 
         """
         return f"AgentConfiguration(id={self.id}, key={self.key}, value={self.value})"
+    
+    @classmethod
+    def update_agent_configurations_table(cls, session, agent_id: Union[int, None], updated_details: AgentRunIn):
```

Comment:
add unit tests for these

Reply:
added a new file test_update_agent_config_table.py